src/components/thumbnail-maker/TagItem.tsx

```
@@ -39,7 +41,18 @@ export function TagItemView({ tag }: { tag: Tag }) {
         "relative flex min-w-fit max-w-full cursor-pointer items-center overflow-hidden truncate whitespace-nowrap rounded-full transition-all duration-300 ease-in-out"
       )}
     >
-      <span>{tag.text}</span>
+      <span>
+        {tag.tagContentType === "3d-emoji" ? (
+          <img
+            src={tag.text.url}
+            width={90}
+            height={90}
+            style={{ pointerEvents: "none" }}
+          />
```

Biome

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.

(lint/a11y/useAltText)